tensorflow/contrib/tensorrt/README.md

```diff
@@ -1,8 +1,50 @@
-# Using TensorRT in TensorFlow
+# Using TensorRT in TensorFlow (TF-TRT)
 
-This module provides necessary bindings and introduces TRT_engine_op operator
-that wraps a subgraph in TensorRT. This is still a work in progress but should
-be useable with most common graphs.
+This module provides necessary bindings and introduces
+`TRTEngineOp` operator that wraps a subgraph in TensorRT.
+This module is under active development.
+
+## Installing TF-TRT
+
+Currently TensorFlow nightly builds include TF-TRT by default,
+which means you don't need to install TF-TRT separately.
+You can pull the latest TF containers from docker hub or
+install the latest TF pip package to get access to the latest TF-TRT.
+
+If you want to use TF-TRT on NVIDIA Jetson platform, you can find
+the download links for the relevant TensorFlow pip packages here:
+https://docs.nvidia.com/deeplearning/dgx/index.html#installing-frameworks-for-jetson
+
+## Installing TensorRT
+
+In order to make use of TF-TRT, you will need a local installation
+of TensorRT. Installation instructions for compatibility with TensorFlow
+are provided on the
+[TensorFlow GPU support](https://www.tensorflow.org/install/gpu) guide.
+
+## Examples
+
+You can find example scripts for running inference on deep learning
+models in this repository: https://github.com/tensorflow/tensorrt
+
+We have used these examples to verify the accuracy and
+performance of TF-TRT. For more information see
+[Verified Models](https://docs.nvidia.com/deeplearning/dgx/integrate-tf-trt/index.html#verified-models).
+
+## Documentation
+
+[TF-TRT documentaion](https://docs.nvidia.com/deeplearning/dgx/integrate-tf-trt/index.html)
```

Comment:
s/documentaion/documentation

Reply:
Thanks @smit-hinsu. Fixed it.

I just copied the doc to a Google doc to make sure I don't have any more typos!